content/deploy-troubleshooting.md

```diff
@@ -4,36 +4,51 @@ order: 15
+<h2 id="general-advice">General Advice</h2>
 
-The 503 error ("Service Unavailable: No healthy enpoints to handle the request") indicates that our image builder was not able to successfully create a container, when attempting to deploy. This usually indicates a problem in your code that prevents deployment. You may also see this error temporarily, if trying to resolve a URL in your browser within 1-2 minutes of issuing your deploy command.
+Check these items if you're having trouble with uptime, performance or deployment.
+- Your Meteor version. More current versions may resolve issues found in older Meteor versions. While the reverse is rare and generally shouldn't happen, if you're recently upgraded Meteor versions and start having difficulties, consider reverting to your last Meteor version.
+- Your container's memory and CPU usage. If your app is running out of memory, you may need to either switch to a larger container or refactor your app to use less memory. In the short term, the only guaranteed solution is to use a container large enough to handle all your app's memory needs.
+- Your app's logs. While 'All' shows all errors, consider breaking it down by tab. If your app is running and struggling before failing, check the 'App' tab. If your app is crashing before deployment and never successfully launching, check the 'Service' tab. Note that both can happen simultaneously: the earlier version of your app may be throwing errors, while the recent version created to fix the problem may have a code error preventing deployment. In that case, 'Service' will be more helpful.
+- Check GitHub to see if your issue is listed with a suggested workaround or solution.
+- Consider running more than 1 container, if uptime for your app is mission-critical and you're currently running a single container.
+- Write in to support. To minimize the back-and-forth, please send in the name of the affected app, the conditions that trigger the issue (confirmed or suspected), steps to reproduce, and relevant logs. Try to resolve errors listed in the logs before writing in.
 
-A common cause of this error is that your `MONGO_URL` variable is missing or is set incorrectly. You can verify what `MONGO_URL` Galaxy is using by going to the app's dashboard and choosing the settings tab. To learn how to set it correctly, check the following resources:
+Note that code-level review lies outside the scope of Galaxy's support.
 
-* [How do I set Environment Variables?](https://galaxy.meteor.com/help/setting-environment-variables) will show you how to set up your `settings.json` file.
+<h2 id="503-errors">503 errors and Deployment failures</h2>
+
+The 503 error will show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. This indicates that our image builder was not able to successfully create a container, when attempting to deploy. This usually indicates a problem in your code that prevents deployment.
```

Comment:
Also this isn't really true. This means we are not currently running any healthy containers.  One reason is that all containers are unhealthy — eg they are all stuck in a CPU loop.

Another reason is that none are running — maybe they all just crashed (say, maybe there's only one in the first place and it just crashed and hasn't quite restarted).

Another reason is that a build failed, but this is only really relevant for your initial deploy, or if your previous version is broken for one of the above reasons. We don't start killing old containers until the new version is built and some new containers are running and healthy.